server/controllers/courses/level/getCoursesLevel.js

```diff
@@ -1,18 +1,18 @@
-const getAvailableCourses = require('../../../database/queries/getAllCourses');
+const getCoursesByLevel = require('../../../database/queries/getCoursesLevel');
 
-exports.getAllCourses = (req, res) => {
-  const { level } = req.body;
-  getAvailableCourses(level)
-    .then((allCourses) => {
-      if (!allCourses.rowCount) {
-        res.send({
+exports.getCoursesLevel = (req, res) => {
+  const { level } = req.params;
+  getCoursesByLevel(level)
+    .then((courses) => {
+      if (!courses.rowCount) {
+        res.status(404).send({
           error: 'No courses available for this level',
```

Comment:
i think it's not an error 
```suggestion
          error: 'No courses available for this level',
```